lib/jira/commands/sprint.rb

```diff
@@ -1,37 +1,37 @@
 module Jira
   class CLI < Thor
 
-    desc "sprint", "Lists all tickets in active sprint"
-    def sprint
-      # TODO
-      self.api(:agile).get(
-        'rapid/charts/sprintreport?rapidViewId=17&sprintId=41'
-      ) do |response|
-        groups = {}
+    #desc "sprint", "Lists all tickets in active sprint"
```

Comment:
:+1: - I'll clean up this logic in a future commit and get it back into the gem then. Can you delete this entire file instead of commenting it out?